src/scenes/Languages/Edit/EditLanguage.tsx

```diff
@@ -0,0 +1,32 @@
+import { useSnackbar } from 'notistack';
+import * as React from 'react';
+import { FC } from 'react';
+import { Except } from 'type-fest';
+import { useUpdateLanguageMutation } from './EditLanguage.generated';
+import {
+  EditLanguageForm,
+  EditLanguageFormProps as Props,
+} from './EditLanguageForm';
+
+type EditLanguageProps = Except<Props, 'onSubmit'>;
+
+export const EditLanguage: FC<EditLanguageProps> = (props) => {
+  const [updateLanguage] = useUpdateLanguageMutation();
+  const { enqueueSnackbar } = useSnackbar();
+
+  const submit: Props['onSubmit'] = async (input) => {
+    const res = await updateLanguage({
+      variables: { input },
```

Comment:
Is the `id` actually included here in form values? (`input`)

Reply:
I checked again to make sure and the id is included in the form values

Reply:
Thanks. That's good news. We'll continue this pattern then for our mutations 💯